mono/metadata/object.c

```diff
@@ -4645,17 +4656,25 @@ int
 mono_runtime_run_main (MonoMethod *method, int argc, char* argv[],
 		       MonoObject **exc)
 {
+	int res;
+
 	MONO_REQ_GC_UNSAFE_MODE;
```

Comment:
nit: Should remove this.  It doesn't make sense to require the thread to already be in GC Unsafe and then immediately switch to GC Unsafe

Reply:
I didn't get around to the nit. :(